Tests/Features/Context/ResourceContext.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\ResourceRestBundle\Tests\Features\Context;
+
+require_once(__DIR__ . '/../../Resources/app/AppKernel.php');
+
+use Behat\WebApiExtension\Context\WebApiContext;
+use Behat\Symfony2Extension\Context\KernelDictionary;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Gherkin\Node\TableNode;
+use PHPCR\Util\NodeHelper;
+use PHPCR\Util\PathHelper;
+
+class ResourceContext extends WebApiContext
+{
+    use KernelDictionary;
```

Comment:
Btw, you aren't using the WebApiContext in this context. You should specify it in the `contexts` setting in `behat.yml` instead of extending it here.